src/components/graph/menus/create-node-menu.js

```diff
@@ -174,7 +174,12 @@ const CreateNodeMenu = ({
             selectionForCopy?.copyType === CopyType.SUBTREE_CUT
         );
     }
-
+    function isSubtreeRemovingAllowed() {
```

Comment:
I suggest a name of function that has relation to check if the parentNode has children. Example: parentNodeHasChildren